README.md

```diff
@@ -76,6 +76,9 @@ export default defineNuxtConfig({
 // Show connect portal
 connector.showConnectPortal();
 
+// Show connect portal with preferred provider (`providerId` is the EIP-6963 RDNS)
+connector.showConnectPortal({ providerId: 'io.metamask' });
```

Comment:
Name it `preferredProviderId` to avoid confusion?

Reply:
Updated